### PR TITLE
Host: add possibility to download file from url

### PR DESCRIPTION
**Pull request by cynepco3hahue**
- download file from url and show runtime progress bar

### CODE REVIEW

File: rrmngmnt/host.py [hunk 1]
```diff
@@ -336,3 +337,41 @@ def is_connective(self, tcp_timeout=20.0):
             ret = False
 
         return ret
+
+    def download_file_from_url(self, url, f_dir):
+        """
+        Download file on the host from given url
+
```
Comment (myakove) [inline rrmngmnt/host.py]: to the host

File: rrmngmnt/host.py [hunk 2]
```diff
@@ -336,3 +337,41 @@ def is_connective(self, tcp_timeout=20.0):
...
+        :param url: url to file
```
Comment (myakove) [inline rrmngmnt/host.py]: File download url

File: rrmngmnt/host.py [hunk 3]
```diff
@@ -336,3 +337,41 @@ def is_connective(self, tcp_timeout=20.0):
...
+        :type url: str
+        :param f_dir: file directory on host
```
Comment (myakove) [inline rrmngmnt/host.py]: Directory where to save the file

**Comment by lukas-bednar** [comment]
I am not sure whether it belongs here ... I would rather to download that file directly on the host instead of downloading that on client and push it to host. 

If you have a good reason I can consider that to include that.

**Comment by cynepco3hahue** [comment]
I want to use this function to download rhevm-appliance(ova or rpm) on host from brew.

File: rrmngmnt/host.py (hunk 3 above)
Comment (cynepco3hahue) [inline rrmngmnt/host.py]: done

File: rrmngmnt/host.py (hunk 2 above)
Comment (cynepco3hahue) [inline rrmngmnt/host.py]: done

File: rrmngmnt/host.py (hunk 1 above)
Comment (cynepco3hahue) [inline rrmngmnt/host.py]: done

**Comment by cynepco3hahue** [comment]
I also think we can use this function to get new rhevh for upgrade

**Comment by cynepco3hahue** [comment]
Have pull request to rhevm-qe-automation

**Pull request closed without merging**